samples/ComponentExamples/src/utils/getEndpointUrl.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+let endpointUrl: string | undefined;
```

Comment:
Don't use global mutable variables.

Let the caller of `getEndpointUrl` cache the value if they want. (and I'd rename this function to `fetchEndpointUrl` to reinforce that there is a `fetch` involved.

Reply:
That's a code piece copy&paste from demo server(I believe that is for caching), but yea, since caching is not necessary for the test, let me edit this part